packages/design-system/src/components/N8nInput/Input.vue

```diff
@@ -58,7 +67,15 @@ const props = withDefaults(defineProps<InputProps>(), {
 	autocomplete: 'off',
 });
 
-const computedSize = computed(() => (props.size === 'xlarge' ? undefined : props.size));
+const resolvedSize = computed(
+	() =>
+		(props.size === 'xlarge' ? undefined : props.size) as
```

Comment:
ElementPlus components have a different type declared. Technically they accept any value for the prop and will generate a class based on it, but `vue-tsc` will complain because it's not the right `PropType`.

Reply:
If it makes sense here or too complicated feel free to keep.